clockwork.go

```diff
@@ -25,6 +26,8 @@ type FakeClock interface {
 	// BlockUntil will block until the FakeClock has the given number of
 	// sleepers (callers of Sleep or After)
 	BlockUntil(n int)
+	// AddTo creates a derived context that references the clock.
+	AddTo(ctx context.Context) context.Context
```

Comment:
I understand this is a convenience change, but it's also a backward incompatible one and I'm not sure it's worth the cost compared to just using `clockwork.AddTo`.